openapi/mx_platform_api.yml

```diff
@@ -1954,6 +2282,55 @@ components:
         pagination:
           "$ref": "#/components/schemas/PaginationResponse"
       type: object
+    MonthlyCashFlowResponse:
+      properties:
+        guid: 
+          example: MCF-4e431124-4a29-abf9-f059-ab232ac14dbf
+          type: string
+          description: Unique identifier for the monthly cash flow profile. Defined by MX.
+        user_guid:
+          example: USR-6c83f63c-efcc-0189-3f14-100f0bad378a
+          type: string
+          description: Unique identifier for the user the monthly cash flow profile is attached to. Defined by MX.
+        budgeted_income: 
+          example: 1200.12
+          type: integer
+          description: The amount of the budgeted income for the user.
+        budgeted_expenses:
+          example: 1000.00
+          type: integer
+          description: The amount of the budgeted expenses for the user.
+        goals_contribution: 
+          example: 150.00
+          type: integer
+          description: The monthly dollar amount allocated for goals.
+        estimated_goals_contribution:
+          example: null
```

Comment:
example: null? I mean it IS nullable, but that is not the most example-y example

Reply:
I think this one is null because the example for `uses_estimated_goals_contribution` is `false`. I think in this case `null` is a reasonable example -y example 😊

Reply:
😎